cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -71,7 +71,7 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
     )
     prepareAndTestQuery(
       title = "Create a unique constraint only if it does not already exist",
-      text = "If it is unknown if a constraint exists or not but we want to make sure it does, we add the `IF NOT EXISTS`. " +
+      text = "If it is not known whether a constraint exists or not, add `IF NOT EXISTS` to ensure it does. " +
```

Comment:
Yes, this works! In general, we try to steer away from using pronouns, like _we_ or _you_.